### DISCUSSION TITLE
Windows Support

### DISCUSSION THREAD

**Discussion by theSuess** [cashapp/hermit#173] [General]
Hey there, we would love to use hermit in our organization, but sadly we have a very heterogeneous IT landscape and would need windows support.

Is someone already working on this, or should we just implement it ourselves and submit a pull request?

**Comment by alecthomas** [comment]
Hi there! Nobody is working on Windows support, but we would love contributions to add it. It's only missing because we don't have any Windows machines.

If you'd like to jump in the Slack channel (linked from the README), we'd be happy to point you in the right direction, answer questions, etc.

**Comment by alecthomas** [comment]
PS. We'd really appreciate the contribution.
